rmc/utils/generic.py

```diff
@@ -349,23 +370,28 @@ def keep_criteria(
     cov_expr: hl.expr.Int32Expression,
     af_threshold: float = 0.001,
     cov_threshold: int = 0,
+    filter_to_rare: bool = True,
 ) -> hl.expr.BooleanExpression:
     """
     Return Boolean expression to filter variants in input Table.
 
     Default values will filter to rare variants (AC > 0, AF < 0.001) that pass filters and have median coverage greater than 0.
 
-    :param hl.expr.Int32Expression ac_expr: Allele count Int32Expression.
-    :param hl.expr.Float64Expression af_expr: Allele frequency (AF) Float64Expression.
-    :param hl.expr.SetExpression filters_expr: Filters SetExpression.
-    :param hl.expr.Int32Expression cov_expr: gnomAD median coverage Int32Expression.
-    :param float af_threshold: Remove rows above this AF threshold. Default is 0.001.
-    :param int cov_threshold: Remove rows below this median coverage threshold. Default is 0.
+    :param ac_expr: Allele count Int32Expression.
+    :param af_expr: Allele frequency (AF) Float64Expression.
+    :param filters_expr: Filters SetExpression.
+    :param cov_expr: gnomAD median coverage Int32Expression.
+    :param af_threshold: Remove rows above this AF threshold. Default is 0.001.
+    :param cov_threshold: Remove rows below this median coverage threshold. Default is 0.
```

Comment:
```suggestion
    :param cov_threshold: Median coverage threshold below which variants are removed. Default is 0.
```

Reply:
GitHub isn't letting me apply this suggestion (and the one above), but I think I prefer the active vs passive voice here

Reply:
Ok I made this suggestion because the original wording makes me initially think of a boolean (remove or not), but it's understandable nonetheless